### DISCUSSION TITLE
关于一键部署的问题，强迫症有点难受

### DISCUSSION THREAD

**Discussion by itcodes** [ChatGPTNextWeb/NextChat#1432] [Q&A]
一键部署的时候，突然多了一个私有项目。自己看了教程，应该是操作问题。现在 2 个项目都能用。一般是用哪个呀？
一个是 fork 的项目，一个是不知道点啥了，多出来的一个项目。想问下这个一般是选择 fork 的项目是吗？我理解是可以检测更新
<img width="646" alt="image" src="https://github.com/Yidadaa/ChatGPT-Next-Web/assets/8282645/16b6b0c8-944d-4e70-b33f-361b31306ca5">

**Comment by Yidadaa** [comment]
建议保留 fork 项目，删掉另外的。
